### PR TITLE
Kriselda/api routes

### PR DESCRIPTION
**Pull request by kbonifacio**
Created products router and routes for the following:

- "/"
- "/wine/:productId"
- "/cheese/:productId"

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 const apiRouter = require('express').Router();
 
+const productsRouter = require("./product");
+apiRouter.use("./product", productsRouter)
```
Comment (hoguer) [inline routes/index.js]: `apiRouter.use("/products", productsRouter);`

File: routes/products.js [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+productsRouter.get("/:productId", async (req, res, next) => {
+    try {
+        const products = await getProductById(id);
+        res.send(products)
+    } catch (error) {
+        throw error
+    }
+});
+
+productsRouter.post("/newproduct", async (req, res, next) => {
```
Comment (hoguer) [inline routes/products.js]: `productsRouter.post("/", async (req, res, next) => {`
```
CRUD:
POST /products
GET /products
GET /products/:productId
PATCH /products/:productId
DELETE /products/:productId
 ```

File: routes/index.js [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+
 const usersRouter = require("./users");
 apiRouter.use("./users", usersRouter)
```
Comment (hoguer) [inline routes/index.js]: `apiRouter.use("/users", usersRouter)`

File: routes/products.js [hunk 4]
```diff
@@ -1,3 +1,42 @@
...
+        const allProducts = await getAllProducts();
+        res.send(allProducts)
+    } catch (error) {
+        throw error
+    }
+} );
+
+productsRouter.get("/:productId", async (req, res, next) => {
+    try {
+        const products = await getProductById(id);
```
Comment (hoguer) [inline routes/products.js]: would it make more sense to call this `product` since it's just one product?

File: routes/index.js (hunk 1 above)
Comment (kbonifacio) [reply routes/index.js]: Edits were implemented.

File: routes/products.js (hunk 2 above)
Comment (kbonifacio) [reply routes/products.js]: Edits were implemented.

File: routes/index.js (hunk 3 above)
Comment (kbonifacio) [reply routes/index.js]: Edits were implemented.

File: routes/products.js (hunk 4 above)
Comment (kbonifacio) [reply routes/products.js]: Edits were implemented.

**Review by Felix-Cadiz — APPROVED** [review]
Thanks for the changes!

**Review by richardedwardbrown3 — COMMENTED** [review]
Looked over everything. Looks good.

**Pull request merged**